src/js/page_sections/Menu/Menu.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import request from 'graphql-request';
+
+import allTopicPagesQuery from 'js/queries/allTopicPagesQuery';
+import I18nNavLink from 'js/modules/I18nNavLink';
+import ExternalLink from 'js/modules/ExternalLink';
+import MenuItem from 'js/page_sections/Menu/MenuItem';
+import navigation from '__tmpdata/navigation';
+
+import CloseSVG from 'js/svg/Close';
+import AirplaneSVG from 'js/svg/Airplane';
+import citySealImg from 'images/coa_seal.png';
+
+
+class Menu extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openSection: null,
+    };
+    this.DESKTOP_BREAKPOINT = 1080;
```

Comment:
made a constants file with breakpoints to share across our js
https://github.com/cityofaustin/janis/blob/fb35ba5e62f834729f833345dfb06d113559e5d1/src/js/helpers/constants.js